pipelines/skylab/multiome/test_inputs/Scientific/10k_pbmc.json

```diff
@@ -1,6 +1,5 @@
 {
 "Multiome.annotations_gtf":"gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf",
```

Comment:
@ekiernan does this look like the right gtf to be using here?

Reply:
@nikellepetrillo we just confirmed yesterday that the modified one would be the appropriate one for optimus, not the intronic.

Reply:
here is a successful run using `gs://broad-gotc-test-storage/Multiome/input/scientific/10k_PBMC_Multiome/introns_modified_gencode.vGencode_V43.primary_assembly.annotation.gtf` https://cromwell.gotc-dev.broadinstitute.org/api/workflows/1/eb702f53-2de8-410a-9036-72ca3a190f3e/metadata